src/components/LoaderSpinner.vue

```diff
@@ -5,23 +5,22 @@
 </template>
 
 <script lang="ts">
-import { defineComponent } from 'vue';
+import { defineComponent, ref } from 'vue';
 import ProgressSpinner from 'primevue/progressspinner';
 
 export default defineComponent({
   name: 'LoaderSpinner',
   components: {
     ProgressSpinner,
   },
-  props: {
-    isLoading: {
-      type: Boolean,
-      required: true,
-    },
-  },
   setup() {
     return {};
   },
+  data() {
+    return {
+      isLoading: this.$isLoading,
+    };
+  },
```

Comment:
If you use option API and "setup" is empty, maybe should be removed "setup"?

Reply:
True true